blockstack_client/client.py

```diff
@@ -77,38 +54,35 @@ def session(conf=None, config_path=CONFIG_PATH, server_host=None, server_port=No
     Returns the API proxy object.
     """
 
-    if conf is None and config_path is not None:
+    if not conf and config_path:
         conf = get_config(config_path)
 
-    if conf is not None:
-        if server_host is None:
-            server_host = conf['server']
-        if server_port is None:
-            server_port = conf['port']
-        if storage_drivers is None:
-            storage_drivers = conf['storage_drivers']
-        if metadata_dir is None:
-            metadata_dir = conf['metadata']
-        if spv_headers_path is None:
-            spv_headers_path = conf['blockchain_headers']
-
-    if storage_drivers is None:
-        log.error("No storage driver(s) defined in the config file.  Please set 'storage=' to a comma-separated list of drivers")
+    if conf:
```

Comment:
Also pointing these out as cases where checking for None explicitly would be preferred (since a lot could be going on behind the scenes here).